requirements.txt

```diff
@@ -7,3 +7,6 @@ pyclesperanto_prototype==0.7.6
 magicgui
 numpy!=1.19.4
 pyperclip
+loguru
```

Comment:
if you like this we can leave it in... was mostly using it temporarily for debugging.  (but it is a lovely library!)
if we want to keep it permanently, we should probably (in a later PR) decrease the verbosity by default

Reply:
Yes, I like it. Need to make myself familiar with proper logging.